sass/_variables.scss

```diff
@@ -17,11 +17,15 @@ $white-transluant:       rgba($white, .8) !default;
 // Brand colors
 // -------------------------
 
-$brand-primary:         #f4d430 !default;
+// Could also be #3EA0DA, depending on HBP identity renewal
+$brand-primary:         #82BBC4 !default; // was F4D430 in v1
```

Comment:
It will changes a lot of properies likely all buttons. It will probably clash with link colors as well as they are blue. Can you just set the new blue to brand-info for now?